### ISSUE TITLE
For Cycle 11

### ISSUE THREAD

**Issue by leo816** [bisq-network/compensation#495]
## Summary

 - BSQ requested: **4100**        
   
    
 

## Contributions delivered

I have mediated **313** finished cases so far. 
16 cases from the old mediation.
83 cases in November
62 on December
69 on January
61 on February
22 on March so far

I request 50 BSQ / mediation case. As discussed with @cbeams this is the last cycle using 50 BSQ/ case (The other mediator was already paid using this price, summing up a total of 8450 BSQ, on my case I will sum up 8200 BSQ) From the next cycle onwards, the price used for compensation per case will be cut in half and mentioned in USD from now on to  **16.5 USD** .
The goal is to eventually have a fixed rate as it would, among other things, ensure the mediator's interests are aligned with Bisq's.

I request 4100 BSQ as suggested on cycle 10 to meet the budget criteria for this cycle and the forthcomings. Will spread out the compensation requests accordingly.

(From cycle 12 onwards, individual cases handled on Keybase that are eligible for compensation will be addressed separately.)

**4100** BSQ correspond to 82 finished cases.
Out of the 313 cases, 82 were already accounted for on cycle 10.

I have also been available on Keybase **everyday** dealing with any issues concerning the mediation tickets I've been assigned to or questions of any kind.
I have also been helping to solve old arbitration disputes or done manual payouts to solve the other mediator's dispute due to the fact he fell sick for a some time. I have not asked for any compensation regarding my work on Keybase as I considered it part of the mediation role per se.

## Contributions in progress

Work goes on daily.
## Roles performed

My summary of my work is given on bisq-network/roles#83
I provide a summary of the main issues that have caused mediation tickets to occur in the first place and the ones that are most important to solve for user's satisfaction.

**Comment by cbeams** [comment]
Review is complete from my side, @leo816. Please see my comments on future Mediator compensation requests at https://github.com/bisq-network/roles/issues/83#issuecomment-598164440.

Feel free to submit your DAO proposal and paste the tx id here when complete, but you might want to give it a little more time before doing so in order to allow additional feedback, since this compensation arrangement has been a topic of conversation.

**Comment by leo816** [comment]
Proposal tx id: 0ccbbfd1bccc944d3f999df8972175f5063dfbbc49e3c97a8965577403c026c5

**Comment by wiz** [comment]
> From the next cycle onwards, the price used for compensation per case will be cut in half to 25 BSQ.

This is not only too high IMO, but getting paid per-case compensation disincentivizes mediators from reporting/fixing bugs because it would reduce their salary but flat-rate compensation incentivizes mediators to report/fix bugs to get their case load down. We should always try to make incentives in Bisq align properly as much as possible.

**Comment by leo816** [comment]
@cbeams and I agreed with the fixed rate here bisq-network/roles#83 .(will make that clear on the proposal as well) I think is the best way going forward, the only issue with me implementing that right now is that since I have been adjusting my compensation requests to meet the budget criteria, I have many mediation cases (around 200+ since mediation started in October) for which I haven't received compensation. As you probably saw in the comments we fixed it at 16.5 USD per case, and when I catch up I would have a flat rate, or at least that was the solution that I found most logical. Would like to hear your thoughts on this.
